File: Easy/LengthOfLastWord/SolutionTest.py
from unittest import TestCase, main
from Solution import Solution

class SolutionTest(TestCase):
    def test_lengthOfLastWord(self):
        self.assertEquals(5, Solution.lengthOfLastWord("Hello World"))


    def test_lengthOfLastWord2(self):
        self.assertEquals(4, Solution.lengthOfLastWord("   fly me   to   the moon  "))


    def test_lengthOfLastWord3(self):
        self.assertEquals(6, Solution.lengthOfLastWord("luffy is still joyboy"))


if __name__ == "__main__":
    main()

File: Easy/LengthOfLastWord/SolutionTest.java
package Easy.LengthOfLastWord;

import org.junit.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class SolutionTest{
    
    @Test
    public void lengthOfLastWordTest(){
        assertEquals(5, new Solution().lengthOfLastWord("Hello World"));
    }

    @Test
    public void lengthOfLastWordTest2(){
        assertEquals(4, new Solution().lengthOfLastWord("   fly me   to   the moon  "));
    }

    @Test
    public void lengthOfLastWordTest3(){
        assertEquals(6, new Solution().lengthOfLastWord("luffy is still joyboy"));
    } 
}
